libs/core/src/internal/kirby-element.ts

```diff
@@ -0,0 +1,12 @@
+import { LitElement } from 'lit';
+
+export default class KirbyElement extends LitElement {
+  static define(name = this.tagName, elementConstructor = this) {
+    const currentlyRegisteredConstructor = customElements.get(name) as CustomElementConstructor;
```

Comment:
Add a console.error and stop function, if 'name' is not provided. This may lead to an exception when trying to define a customElement that is undefined.

Reply:
Added check for `tagName` 👍🏻